modules/creating-a-machine-pool-cli.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="creating_machine_pools_cli_{context}"]
+= Creating a machine pool using the ROSA CLI
+
+You can create additional machine pools for your {product-title} (ROSA) cluster by using the ROSA CLI (`rosa`).
+
+[IMPORTANT]
+====
+The compute node instance types, autoscaling options, and node counts that are available to you depend on your ROSA subscriptions, resource quotas and deployment scenario. For more information, contact your sales representative or Red Hat support.
+====
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a ROSA cluster.
+
+.Procedure
+
+* To add a machine pool that does not use autoscaling:
```

Comment:
This step and the next unordered list item look like they each only have technically a one-step procedure "under" them as substeps. Could each of the unordered list items just absorb their respective substeps? So it's basically just down to 2 unordered items w/ no substeps.

Reply:
@adellape thank you for your review and for this suggestion. Resolved.